.github/workflows/pr-plugin-build.yml

```
@@ -0,0 +1,199 @@
+          cd ..
+          
+          # Generate file list for plugin
+          find build/usr/local/emhttp -type f | sed 's|^build||' > file_list.txt
+      
+      - name: Configure AWS CLI for R2
+        if: steps.changed-files.outputs.has_changes == 'true'
+        run: |
+          aws configure set aws_access_key_id ${{ secrets.CLOUDFLARE_PREVIEW_ACCESS_KEY_ID }}
+          aws configure set aws_secret_access_key ${{ secrets.CLOUDFLARE_PREVIEW_SECRET_ACCESS_KEY }}
+          aws configure set region auto
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)